lib/gap_intelligence/models/header.rb

```diff
@@ -0,0 +1,13 @@
+module GapIntelligence
+  class Header < Record
+    attribute :name
+
+    def initialize(attrs = {})
```

Comment:
do we need to overwrite `initialize`? I though `@raw` keep original raw values as it, `@id` will be nil anyway and `@attributes` will be processed accordingly from whatever is defined as attribute, imho 👀